compat/src/index.d.ts

```diff
@@ -47,6 +45,18 @@ export declare interface Children {
   toArray: (children: preact.ComponentChildren) => preact.VNode<{}>[];
 }
 
+//
+// Suspense/lazy
+// -----------------------------------
+function lazy<T>(loader: () => Promise<{default: T}>): T;
```

Comment:
This should probably be `export declare function lazy...` to fall in line with the rest.